recipes/tbl2asn/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+    name: tbl2asn
+    version: "25.0"
+
+source:
+    fn: mac.tbl2asn.gz # [osx]
+    url: ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/mac.tbl2asn.gz # [osx]
+    sha256: d18785994de6e20e7f3502aa042b59817eaf58365b11e2b32b95856bc4738ec1 # [osx]
+    fn: linux.tbl2asn.gz # [linux]
+    url: ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/linux.tbl2asn.gz # [linux]
+    sha256: 0f95c6c2872b2beb908eb42a64b15ed282424332faab3c442b43fc8b99548e58 # [linux]
+    fn: linux64.tbl2asn.gz # [linux64]
+    url: ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/linux64.tbl2asn.gz # [linux64]
+    sha256: 0f95c6c2872b2beb908eb42a64b15ed282424332faab3c442b43fc8b99548e58 # [linux64]
+
+build:
+    number: 0
+
+# Currently only the binaries are available, and they use GLIBC_2.7 on Linux,
+# making them incompatible with our GLIBC_2.5-based CentOS 5 test environment
+test:
+    commands:
+        - tbl2asn --help | grep arguments # [osx]
```

Comment:
I'm not wild about skipping tests under Linux, but if the alternative is no tbl2asn package we can probably live with it. Can we test executability post-install via a `post-link.sh` script?  If the script fails with a non-zero exit code, no package metadata is written, and the package is not considered installed. It may have to check for some environment variable to determine if it is being called during build vs. install (skip the tbl2asn call if build, run if install).